### ISSUE TITLE
Moderator cannot unpin himself or herself from thumbnail window menu

### ISSUE THREAD

**Issue by mabocoglu** [jitsi/jitsi-meet#11291]
### Description:

In a conference with multiple participants, moderator cannot unpin himself or herself even he or she click on unpin from the thumbnail menu on the web. I did not try from mobile.

### Steps to reproduce:

1. Create a conference
2. Use jwt token to login for each participant
3. Make sure you are the moderator
4. Add some partiticipant
5. While you are moderator select one of participants while the conference in tileview mode
6. You see the participant will be pinned
7. Try to unpin yourself from the thumbnail window

### Expected behavior:
Moderator can unpin himself or herself when he or she click on unpin command from the thumbnail window.
That behavior does not change when moderator left conference and the participant who get the moderator affiliation. The new moderator also cannot unpin himself or herself.

### Actual behavior:

Nothing changed on unpin command of the moderator thumbnail window

### Server information:

- Jitsi Meet version: 2.0.742-1 unstable
- Operating System: Ubuntu 20.04.4 LTS AMD

### Client information:

- Browser / app version: Google Chrome 100.0.4896.60
- Operating System: Windows 10 21H2 x64

### Additional information:

JWT is enabled for conference login.  

For regular users there is no such a problem. He or she can unpin himself or herself.

**Comment by saghul** [comment]
I don't think JWT auth has anything to do with it.

Can you reproduce this on meet.jit.si? I can't.

Note that we removed the pinned user indicator and will be bringing it back soon, maybe is the lack of an indicator that is confusing? Clicking on the tile again will unpin the user.

**Comment by mabocoglu** [comment]
I don't think jwt does not effect too. I just gave as a detail.

on meet.jit.si there is older version of jitsi-meet due to the lack of pin option in the thumbnail menu.

In my situation there is a newer version than meet.jit.si

In addition I can't see a moderator mode on meet.jit.si. It looks like everyone is moderator. Everyone can kick any other participant here. Anyway there is no pin participant or unpin participant option there. Thus we don't deal with it I think.

**Comment by mabocoglu** [comment]
I have also test by switching to tileview again. Nothing changed. (Pinning and unpinning are available on filmstrip mode. Sorry)

**Comment by saghul** [comment]
> In my situation there is a newer version than meet.jit.si

Ah, gotcha. Then you can try to repro on https://alpha.jitsi.net which is rebuilt after each commit.



> In addition I can't see a moderator mode on meet.jit.si. It looks like everyone is moderator. Everyone can kick any other participant here.

Yes, everyone is a moderator in meet.jit.si.

**Comment by mabocoglu** [comment]
it's working on alpha.jitsi.net. Is there any difference with jitsi-meet_7142-1 ? In my installation, websocket, sourceNameSignaling and sendMultipleVideoStreams enabled. In jvb.conf multi-stream also enabled.

I have also purged everything and installed jitsi-meet_7142-1. Without any modification in configurations (no jwt, websocket etc), pinning and unpinning is not working very well. 

I have opened a conference with three participants.

First participant became moderator which is set by jitsi-meet (not with jwt). This moderator cannot unpin 2nd participant,

Second participant is a normal participant and cannot unpin 3rd participant.

Third participant is a normal participant and cannot unpin himself or herself.

I am confused. This users joined conference respectively. There should be a relation between unpinning problem with automatic pinning I think.
